Makefile

```diff
@@ -40,6 +41,18 @@ export VSOCK=1
 CARGO_OSDK_ARGS += --init-args="/test/run_vsock_test.sh"
 endif
 
+ifeq ($(LOG_LEVEL), error)
+CARGO_OSDK_ARGS += --kcmd-args="loglevel=0"
```

Comment:
Rename the kernel parameter to `ostd.log_level`. This avoids potential name conflicts if the kernel on top of ostd decides to use "log_level" to specify its own level of its logging facility. Adding the `_` separator makes the parameter name more consistent with Rust style. In addition, the use of `.` as the separator is consistent with Linux's kernel parameter style.